link_shortener/core/filter.py

```diff
@@ -0,0 +1,23 @@
+'''
+Copyright (C) 2020 Link Shortener Authors (see AUTHORS in Documentation).
+Licensed under the MIT (Expat) License (see LICENSE in Documentation).
+'''
+
+
+def filter_links(data, filters):
+    owners = set(row['owner'] for row in data)
+
+    if filters['owner']:
+        data = [row for row in data if row['owner'] == filters['owner']]
+
+    if filters['search']:
+        data = [row for row in data if filters['search'] in row['endpoint']]
```

Comment:
Proč 'in'?

Reply:
Mame hladat iba v tomto stlpci, preto to obmedzenie.

Reply:
To vlastně chodí jako nějaké divné arraye s jedním elementem co?